### ISSUE TITLE
undefined reference to `geodesy::toMsg(geodesy::UTMPose const&)'

### ISSUE THREAD

**Issue by Roboterbastler** [ros-geographic-info/geographic_info#36]
Reproducible e.g. by adding this unit test to `test_utm.cpp`:

```cpp
TEST(UTMPose, toMsg)
{
  double e = 1000.0;
  double n = 2400.0;
  double a = 200.0;
  uint8_t z = 14;
  char b = 'R';
  geodesy::UTMPoint pt(e, n, a, z, b);

  // identity quaternion
  geometry_msgs::Quaternion q;
  q.x = 1.0;
  q.y = 0.0;
  q.z = 0.0;
  q.w = 0.0;
  geodesy::UTMPose pose1(pt, q);
  EXPECT_TRUE(geodesy::isValid(pose1));

  geographic_msgs::GeoPose gp = geodesy::toMsg(pose1);
}
```

**Comment by SteveMacenski** [comment]
PRs are welcome implementing any changes required

**Comment by Roboterbastler** [comment]
Was fixed by #37
